src/navigation/RootNavigator.tsx

```diff
@@ -75,22 +78,186 @@ export default function RootNavigator() {
             <Stack.Screen
               name="TabsStack"
               component={TabsStack}
-              options={{ headerShown: false }}
+              options={({ navigation }) => ({
+                headerTitle: () => {
```

Comment:
[TO DO]: If possible, move code blocks that you use multiple times into their own functions to reduce duplicate code. For instance, the `headerRight` and `headerLeft` function calls render the exact same content, but is duplicated 4x. Good news, you already did this for `LogoTitle`!

Reply:
will do!